File: app/src/main/java/com/wipro/android/proficiencyexercise/netcomm/ServiceAPI.java
package com.wipro.android.proficiencyexercise.netcomm;

import com.wipro.android.proficiencyexercise.model.CanadaList;

import io.reactivex.Observable;
import retrofit2.Call;
import retrofit2.http.GET;

public interface ServiceAPI {

    @GET(ApiConstant.END_POINT)
    Observable<CanadaList> getCanadaList();
}

File: app/src/main/java/com/wipro/android/proficiencyexercise/data/remote/api/ServiceAPI.kt
package com.wipro.android.proficiencyexercise.data.remote.api

import com.wipro.android.proficiencyexercise.data.remote.response.CanadaList
import com.wipro.android.proficiencyexercise.utils.Constants
import io.reactivex.Observable
import retrofit2.http.GET

interface ServiceAPI {

    @get:GET(Constants.END_POINT)
    val canadaList: Observable<CanadaList>
}